common/bloodlines/wc_bloodlines.txt

```diff
@@ -389,4 +389,18 @@ ymiron = {
 	allow_bastards = yes
```

Comment:
Mudclaw dynasty is too old.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/85923552-fac75d80-b89c-11ea-8c2f-950aafe1ae48.png)

</details>